sdk/search/azure-search-documents/azure/search/documents/_service/_index.py

```diff
@@ -0,0 +1,249 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from .edm import Collection, ComplexType
+from ._generated.models import Field
+
+if TYPE_CHECKING:
+    from typing import Any, Dict, List
+
+__all__ = ("ComplexField", "SearchableField", "SimpleField")
+
+
+def SimpleField(
+    name,
+    type,  # pylint:disable=redefined-builtin
+    key=False,
+    filterable=False,
+    facetable=False,
+    sortable=False,
+    hidden=False,
+):
+    # type: (str, str, bool, bool, bool, bool, bool, **Any) -> Dict[str, Any]
```

Comment:
`**Any`?

Reply:
Why is this passing mypy?

Reply:
seems like we did not hard enable mypy check - it's going to pass no matter what.